lms/djangoapps/discussion_api/views.py

```diff
@@ -144,6 +144,9 @@ class ThreadViewSet(_ViewMixin, DeveloperErrorViewMixin, ViewSet):
         * following: If true, retrieve only threads the requesting user is
             following
 
+        * view: "unread" for threads the requesting user has not read, or
```

Comment:
Hmm... `view_filter` might be better in that it's explicit that it's a filter, whereas `view` doesn't make that clear. Unfortunately I can't think of anything else that connects `unanswered` and `unread`.